pg_search/src/postgres/storage/linked_items.rs

```diff
@@ -324,8 +324,7 @@ impl<T: From<PgItem> + Into<PgItem> + Debug + Clone + MVCCEntry> LinkedItemList<
             }
 
             while offsetno <= max_offset {
-                if let Some((deserialized, _)) = page.read_item::<T>(offsetno) {
-                    let PgItem(item, size) = deserialized.into();
+                if let Some(PgItem(item, size)) = page.read_item(offsetno) {
```

Comment:
This was the impetus for this PR.  I realize that it still says `page.read_item()`, but now it's an opaque `PgItem`, not a fully deserialized `T`.